modules/express/src/clientRoutes.ts

```diff
@@ -1449,6 +1545,21 @@ export function setupAPIRoutes(app: express.Application, config: Config): void {
   app.use('/api/v2/user/*', parseBody, prepareBitGo(config), promiseWrapper(handleV2UserREST));
   app.use('/api/v2/:coin/*', parseBody, prepareBitGo(config), promiseWrapper(handleV2CoinSpecificREST));
 
+  app.post(
+    '/api/network/v1/enterprises/:enterpriseId/clients/connections',
+    parseBody,
+    prepareBitGo(config),
+    promiseWrapper(handleNetworkV1EnterpriseClientConnections)
+  );
+
+  app.use(
+    '/api/:namespace/v[12]/enterprises/:enterpriseId/*',
+    parseBody,
+    prepareBitGo(config),
+    promiseWrapper(handleMicroserviceReq)
+  );
+  app.use('/api/:namespace/v[12]/*', parseBody, prepareBitGo(config), promiseWrapper(handleMicroserviceReq));
```

Comment:
This is a dangerous catch all. A lot of express routes contain private keys and wallet password which we do NOT want coming to wallet platform. This makes it much easier for clients to accidentally leak their data if they for ex mistype a url between express and WP.

Reply:
Why do we need to introduce this change in this PR?

Reply:
As well as:
```
  app.use('/api/v2/user/*', parseBody, prepareBitGo(config), promiseWrapper(handleV2UserREST));
  app.use('/api/v2/:coin/*', parseBody, prepareBitGo(config), promiseWrapper(handleV2CoinSpecificREST));
```

Reply:
We could move them into:
```
  // everything else should use the proxy handler
  if (!config.disableProxy) {
  ```

Reply:
Could move into the `disableProxy` function. This would the use of `disableProxy` as well.

Reply:
Update: my code was flawed, so I was able to remove the added middleware and move the namespace catch into disableProxy.